### PR TITLE
newsweek magazine clone

### PR DESCRIPTION
**Pull request by che30**
I implemented the following 
- The navbar 
- The different sections 
- The footer . All these using bootstrap framework

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 
Hi che30
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).
##
 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] please remove commented code. 
- [ ] Remove `!important `from **main.css** file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [ ] After investigation from team, it appeared that your CSS is not linked properly to html. Please correct it. 

So instead of
```
 <link rel="stylesheet" href="/assets/js/bootstrap.min.js">
 <link rel="stylesheet" href="/assets/stylesheets/main.css">
```
It should be
```
 <link rel="stylesheet" href="./assets/js/bootstrap.min.js">
 <link rel="stylesheet" href="./assets/stylesheets/main.css"> 
```

- **When you have made the specified changes, please submit another code review. Thank you!**

Regards,
Arslan Bisharat 
slack : arslanbisharat

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @che30
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##
For this project, we have to make sure nothing is messed up. Kindly work on the suggested change  👇.
- [ ] The project is working fine locally but not online. Please Correct it. 
Instead of  `<link rel="stylesheet" href="/assets/stylesheets/main.css">`
It should be `<link rel="stylesheet" href="./assets/stylesheets/main.css">`

##

- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @che30  . Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [OPTIONAL] :large_blue_circle: 

- [ ] The Twitter account appears as suspended, please change it to a working account or delete from the README.

- [ ] The license link is broken in the README.

- [ ] The website is using the Bootstrap CDN, an the css directory files are not needed anymore. Please remove them

- [ ] There should be one css directory with the main or all needed stylesheets. 

- [ ] All images should have an alternative text atributte. Please make sure all `````` tags have an ```alt``` attribute with a short and meaningful description of the picture.

- [ ] The page should have one ```<h1>``` tag. This helps SEO.

- [ ] There is no significant difference in the 983px and 985px please choose one and merge them together

- [ ] In the "in the magazine" section one element is different than the others but is not appearing in the clone.
image


- [ ] The "editor pick" section elements are missing description and category in the image corner
image
image

- [ ] Footer doesn't look like the original
image

- [ ] Category section doesn't look as expected in tablet view
image

- [ ] Newsletter section doesn't look as expected, the text alignment and the margins of the input. in tablet view
image

- [ ] Footer doesn't look as expected in tablet view
image

- [ ] Navbar doesn't look as expected in mobile view

- [ ] The order of the sections in mobile view is not correct. The original starts with top story, the clone with featured stories

- [ ] Image not rendering 
image

- [ ] More stories element doesn't match original, image is overlaping and text is not surrounding the image
image

- [ ] Newsletter element is not centered and is going out of the website boundaries
image

- [ ] Subscribe section doesn't look like original on mobile version
image

- [ ] Footer doesn't look as expected 
image


## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by Rhelli — CHANGES_REQUESTED** [review]
# Changes Required 🛑♻️

Hi @che30, congratulations on the project so far, you've managed to create an excellent clone and your documentation and Readme is likewise, excellent. There are however a few changes that need to be made in order to get this project approved. If you correct these errors, we'll be able to get you approved and onto the next project!

## 📝  Readme Changes 

**[Mandatory]**  - Please remove the incorrect title and description for the NextWeb project as seen below 👇🏽

image


**[Mandatory]** -  Please make the description to your project in your Readme (the text that appears directly beneath your title) more descriptive. There's a lot to say about this project and you've done a lot to create it, so tell everyone about it! 

I'd suggest summarising the process you went through to create the project, the technologies you used and the overall objectives of the project as a starter. Remember - this is the first thing people see when they look at your project - your goal is to keep them interested enough to look further into your project! 👀



**[Mandatory]** -  The screenshot is excellent and I always recommend it. However I would only screenshot a portion, not the entire page. At the moment your Readme is mostly screenshot! 


**[Optional]** - If you want to show off your project quickly, make a gif and attach it to you Readme. It's more compact (space-wise), more interactive and more eye catching. If you want to know more, [have a look at this tutorial on adding gifs to your Readme.](https://medium.com/@josephcardillo/how-to-add-gifs-to-your-github-readme-89c74da2ce47)


**[Mandatory]** -  There previous TSE addressed the fact that your Readme currently has a broken License link at the end. Please fix this by adding a `LICENSE.txt` file to your Readme. Use [this website](https://choosealicense.com/) to copy and paste a standard license. Easy as pie! 🥧


**[Optional]** -  I wouldn't recommend adding yourself as an Acknowledgement after you've already listed yourself as the author. They're the same thing!

*******

## 👨🏽‍💻 Code Changes

**[Mandatory]** - The previous TSE addressed the issue of none of your links (`<a href>`) having the required `alt` tags. Please add them with a brief description to all of your links. Being able to create accessible HTML is a sought after skill, so best to learn it!



**[Mandatory]** -  The previous TSE pointed out that you should only have one `h1` tag in your HTML and he is completely right. I can currently see 3 different `h1` tags so please downgrade 2 of these to `h2` and reserve the most important title of the page for `h1`.



**[Mandatory]** -  Please amend your 'In The Magazine' section 👇🏽

image

There's two issues to point out here
 1. Please remove the newsline that appears beneath the 'See All Features' section. It does not appear in the original.

2. Please fix the images - they are the wrong height and it is creating a large amount of whitespace which should not be present. I suspect this has happened because you are using the wrong images, which are the wrong ratio and aren't tall enough. To fix this, just download the new NextWeb images (or use the url links) and this section should appear more compact.

For reference, this is what the 'In The Magazine' section should look like 👇🏽

image




**[Mandatory]** -  As suggested by the previous TSE, your 'Editors Pick' section is still missing the topic/category in the bottom right hand corner of the image. This **also** occurs on small screen sizes. Here is your version 👇🏽

image

Here is what it should look like 👇🏽

image




**[Mandatory]** - Please fix your footer. You're nearly there with it, there's just a few tweaks to make to make it perfect! Currently, your footer looks like this 👇🏽

image

It should look closer to this 👇🏽

image

How to fix it? Easy!
1. Give the 'Editions' and 'About Us | Corrections.. etc' divs `padding` on the sides.
2. Give your 'Editions' countries text (i.e. 'U.S. Edition | Pakistan ... etc') a `margin-right` property.
3. Give the '© 2020 NEWSWEEK DIGITAL LLC' element a property of `margin-top: auto` to align it with the 'Newsweek' logo.
4. Check your LinkedIn logo - its different from the rest of the logos in size and color.



**[Mandatory]** - I'm afraid your categories on tablet size screens still need a little work - your not far off though! Here is your version 👇🏽

image
 
The issue here is the `font-sizes` are too large, particularly for the images on the right hand side. Reduce the font sizes and increase the image sizes. The LinkedIn logo **also dissapears** on tablet screen sizes. For reference, this is what it should look like

image

******


Thats it! Get those errors fixed and you'll be onto the next project in no time. Again, excellent work on the project so far. You've managed to create the clone with a minimal amount of custom CSS, your CSS is **super** clean and your HTML is well organised! Make these changes and you'll have a stellar project for your portfolio! 🌟 Good luck!

If you need any help, @ me on Github or feel free to reach out to me via Slack.

All the best,

💂🏽‍♂️ Rory👀

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
 # Newsweek-clone
 A clone of the newsweek magazine for microverse
+# The next web
+
+# A responsive web design based on The newsweek website made for learning purposes.
+## The purpose of the project is to test microverse students mastery of the bootsrap framework and ensure they build responsive webpages across the following viewports. Large screens(laptots) medium screens(tablets) and small screens(Some some smart phones)
```
Comment (descholar-ceo) [inline README.md]: Here you can just use usual text, there is no need to make it a title, since you already have a title

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+- GitHub: [@che30](https://github.com/che30)
+- LinkedIn: [Che Blanchard](https://www.linkedin.com/in/che-nsoh-9455271b0/)
+
+
+## Acknowledgements
+
+**Che Blanchard**
+
+- GitHub: [@che30](https://github.com/che30)
+- LinkedIn: [Che Blanchard](https://www.linkedin.com/in/che-nsoh-9455271b0/)
```
Comment (descholar-ceo) [inline README.md]: Since we have other acknowledgement below, I would recommend you to get rid of this one

File: assets/stylesheets/main.css [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+}
+
+.middle-nav {
+  margin-top: 100px;
+  position: relative;
+  font-weight: 900;
+}
+
+.border-bottom {
+  border-bottom: 1px solid#f4f5f7;
```
Comment (descholar-ceo) [inline assets/stylesheets/main.css:84]: I guess that here you forgot to add some space between `solid#f4f5f7`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1763 @@
...
+                    </h6>
+                  </div>
+                </div>
+              </article>
+              <article>
+                <div class="row border-bottom pb-3 pt-3">
+                  <div class="col-lg-3 col-md-5 ">
+                    <img
+                      src="https://d.newsweek.com/en/full/1653541/donald-trump.jpg?w=95&h=95&f=c77011e270d209e5014af0b7c8a70a88"
+                      class="width-100" alt="	https://d.newsweek.com/en/full/1654388/libresco.png?w=63&h=63&f=d5f8d9cc3f1ca8e1c176e6d9344905f4">
```
Comment (descholar-ceo) [inline index.html]: I would recommend you to use the descriptive `alt` on `` elements as it is suggested with other TSEs

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1763 @@
...
+                    </h6>
+                  </div>
+                </div>
+              </article>
+              <article>
+                <div class="row border-bottom pb-3 pt-3">
+                  <div class="col-lg-3 col-md-5">
+                    <img
+                      src="https://d.newsweek.com/en/full/1653541/donald-trump.jpg?w=95&h=95&f=c77011e270d209e5014af0b7c8a70a88"
+                      class="width-100" alt="	https://d.newsweek.com/en/full/1654388/libresco.png?w=63&h=63&f=d5f8d9cc3f1ca8e1c176e6d9344905f4">
```
Comment (descholar-ceo) [inline index.html]: I would recommend you to use the descriptive `alt` on `` elements as it is suggested with other TSEs

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1763 @@
...
+                        Activists Say</a></h6>
+                  </div>
+                </div>
+              </article>
+              <article>
+                <div class="row border-bottom pb-3 pt-3">
+                  <div class="col-lg-3 col-md-5">
+                    <img
+                      src="	https://d.newsweek.com/en/full/1651856/belarus-security-forces.jpg?w=397&h=265&f=48f083d3661f2c1c0f3ce26747916330"
+                      class="width-100" alt="	https://d.newsweek.com/en/full/1654388/libresco.png?w=63&h=63&f=d5f8d9cc3f1ca8e1c176e6d9344905f4">
```
Comment (descholar-ceo) [inline index.html]: I would recommend you to use the descriptive `alt` on `` elements as it is suggested with other TSEs

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change required :warning: :recycle:
## :wave: :wave: :wave: Hello @che30  :vertical_traffic_light: You are almost there with this big project.
You have done a great job so far, congratulations on that.
- I liked that you used some semantic HTML,
- And that you tried your best to use CSS best practices, keep that up :muscle: :muscle:

However you need to fix few things to make it on the standard.
### The comment marked as __[Required]__ must be implemented to get the project approved, the other marked as __[Optional]__ aren't compulsory to get this project approved.

## Required changes :red_circle:
- [x] As it is mentioned with the previous TSEs, I would recommend you to use descriptive `alt` attributes on all `` elements. Currently you are using links on them, which is not good. Use something like ` alt="name of a user`
- [x] Make sure that your footer is properly fixed as mentioned by the previous TSEs
Clone: 
Screenshot from 2020-10-20 20-57-59
Original: 
Screenshot from 2020-10-20 20-59-38

- [x] Fix the subscribe section because it doesn't look like original on mobile version
Your version: 
Screenshot from 2020-10-20 21-13-09

Original version:
Screenshot from 2020-10-20 21-13-47

- [x] The newsletter section is almost done, but it nees some spacing around the form, 
Your version: 
Screenshot from 2020-10-20 21-16-51
Original version: 
Screenshot from 2020-10-20 21-18-39

## Optional changes :large_blue_circle:
- [ ] The pictures that are on more stories section, make them abit big, 
Your version:
Screenshot from 2020-10-20 21-21-08

Original version:
Screenshot from 2020-10-20 21-22-36

- [x] I noticed that on mobile screens your navbar needs a bit of work
currently it looks like this: 
Screenshot from 2020-10-20 21-05-50
It should look like this:
Screenshot from 2020-10-20 21-06-31


- [x] As requested by the previous TSEs, it would be better if you remove yourself from the acknowledgement section

- [x] Read all other comments I added directly in your codes

:point_right: Make all the above requested changes, and submit an other code review request

If you have something which appears to you as unclear, don't hesitate to reach me out on slack as `@Mugirase Emmanuel` | [LinkedIn](https://www.linkedin.com/m/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by: Mugirase Emmanuel (descholar)

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change required :warning: :recycle:
## :wave: :wave: :wave: Hello @che30 :vertical_traffic_light: Your progress is very good and you are very close to the project approval
You have done a great job so far, congratulations on that.
:+1: You fixed the navbar on mobile, that is very good, I like it.
### However you need to implement all the required changes as they are required in the previous reviews. Kindly check all of them and implement them one by one untill you make it.

:point_right: Make all the above requested changes, and submit an other code review request

:pager: If you have something which appears to you as unclear, don't hesitate to reach me out on slack as @Mugirase Emmanuel | [LinkedIn](https://www.linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Happy coding!

File: README.md (hunk 1 above)
Comment (che30) [reply README.md]: All changes checked above have been implemented as required

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @che30 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] You have repeated the description in the README file two times. Please remove the following text from the README file. The next description is awesome. (Please visit the [image](https://user-images.githubusercontent.com/29689273/96719786-4f01ca80-13c3-11eb-8413-d3387f6ca129.png))

```
## The next web
## A responsive web design based on The newsweek website made for learning purposes.
```
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by che30** [comment]
> # Status: REQUIRED CHANGES  
> Hi @che30
> Good job with the changes!  We are almost there.
> We, however, do have a few changes to carry out before we proceed (see review comment).
> 
> For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below  and work on the suggested changes.
> 
> * [ ]  You have repeated the description in the README file two times. Please remove the following text from the README file. The next description is awesome. (Please visit the [image](https://user-images.githubusercontent.com/29689273/96719786-4f01ca80-13c3-11eb-8413-d3387f6ca129.png))
> 
> ```
> ## The next web
> ## A responsive web design based on The newsweek website made for learning purposes.
> ```
> 
> * **When you have made the specified changes, please submit another code review. Thank you!**
> 
> **If you are stuck at any point or have any questions, let me know on slack at any time. I will be happy to help**
> HAPPY CODING 
> **[Umair Arshad](https://umairarshad.me/)**
> [Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)
Deleted redundant text in readme

**Review by jubaan — APPROVED** [review]
# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

#### :clap: :clap: Great job :hammer_and_pick:!! There are no more issues left to be addressed. 

### [Observations]

- [ ] There are some minor stylelint issues, please fix them before merging.

## See you at the top :rocket:!

# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

**Pull request merged**